### PR TITLE
Drop useless IE8 rules

### PR DESCRIPTION
**Pull request by MorrisJobke**
Noticed while working on the settings page

cc @mar1u5 @blizzz @schiessle @LukasReschke @mikebabb

### CODE REVIEW

**Comment by mar1u5** [comment]
Ahhh... I have missed that we've dropped IE8 in the past :grin: Looks good ... :+1:
Backport?

**Comment by MorrisJobke** [comment]
> Backport?

No real need, because it only reduces the code base but doesn't fix anything.

**Comment by blizzz** [comment]
:fire: hahaha :+1:

File: settings/templates/personal.php [hunk 1]
```diff
@@ -121,11 +121,9 @@
 	<div class="hidden icon-checkmark" id="password-changed"></div>
 	<div class="hidden" id="password-error"><?php p($l->t('Unable to change your password'));?></div>
 	<br>
-	<label for="pass1" class="onlyInIE8"><?php echo $l->t('Current password');?>: </label>
 	<input type="password" id="pass1" name="oldpassword"
 		placeholder="<?php echo $l->t('Current password');?>"
 		autocomplete="off" autocapitalize="off" autocorrect="off" />
-	<label for="pass2" class="onlyInIE8"><?php echo $l->t('New password');?>: </label>
```
Comment (LukasReschke) [inline settings/templates/personal.php]: @Henni Says this is wrong. 🙈 Didn't check but believe him. 🔪

Comment (Henni) [inline settings/templates/personal.php]: usability-wise this should be changed to `class="visually-hidden"`

Comment (MorrisJobke) [inline settings/templates/personal.php]: Ah right. I will create a fix for this. @Henni Thanks for spotting

Comment (MorrisJobke) [inline settings/templates/personal.php]: #122

**Pull request merged**